docs/source/AdministratorGuide/Systems/DataManagement/index.rst

```diff
@@ -43,92 +43,125 @@ DIRAC provides an abstraction of a SE interface that allows to access different
+        }
+      }
+    }
+    StorageElements
+    {
+      CERN-DST-EOS
+      {
+        BaseSE = CERN-EOS
+      }
+      CERN-USER
+      {
+        BaseSE = CERN-EOS
+        PledgedSpace = 205
+        AccessProtocol.1
+        {
+          PluginName = GFAL2_SRM2
+          Path = /eos/lhcb/grid/user
+          SpaceToken = LHCb_USER
+        }
+      }
+      GFAL2_XROOT
       {
-        Host = srm-eoslhcb.cern.ch
+        Host = eoslhcb.cern.ch
         Port = 8443
-        PluginName = GFAL2_SRM2
-        Protocol = srm
-        Path = /eos/lhcb/grid/prod
+        Protocol = root
+        Path = /eos/lhcb/grid/user
         Access = remote
         SpaceToken = LHCb-EOS
         WSUrl = /srm/v2/server?SFN=
       }
-   }
-   CERN-USER
-   {
-      BackendType = Eos
-      SEType = T0D1
-      PledgedSpace = 205
-      AccessProtocol.1
+    }
+
+
+This definition would be strictly equivalent to::
+
+    StorageElementBases
+    {
+      CERN-EOS
+      {
+        BackendType = Eos
+        SEType = T0D1
+        AccessProtocol.1
+        {
+          Host = srm-eoslhcb.cern.ch
+          Port = 8443
+          PluginName = GFAL2_SRM2
+          Protocol = srm
+          Path = /eos/lhcb/grid/prod
+          Access = remote
+          SpaceToken = LHCb-EOS
+          WSUrl = /srm/v2/server?SFN=
+        }
+      }
+    }
+    StorageElements
+    {
+      CERN-DST-EOS
       {
-        Host = srm-eoslhcb.cern.ch
+        BackendType = Eos
+        SEType = T0D1
+        AccessProtocol.1
+        {
+          Host = srm-eoslhcb.cern.ch
+          Port = 8443
+          PluginName = GFAL2_SRM2
+          Protocol = srm
+          Path = /eos/lhcb/grid/prod
+          Access = remote
+          SpaceToken = LHCb-EOS
+          WSUrl = /srm/v2/server?SFN=
+        }
+      }
+      CERN-USER
+      {
+        BackendType = Eos
+        SEType = T0D1
+        PledgedSpace = 205
+        AccessProtocol.1
+        {
+          Host = srm-eoslhcb.cern.ch
+          Port = 8443
+          PluginName = GFAL2_SRM2
+          Protocol = srm
+          Path = /eos/lhcb/grid/user
+          Access = remote
+          SpaceToken = LHCb_USER
+          WSUrl = /srm/v2/server?SFN=
+        }
+      }
+      GFAL2_XROOT
+      {
+        Host = eoslhcb.cern.ch
         Port = 8443
-        PluginName = GFAL2_SRM2
-        Protocol = srm
+        PluginName =  GFAL2_XROOT
+        Protocol = root
         Path = /eos/lhcb/grid/user
         Access = remote
-        SpaceToken = LHCb_USER
+        SpaceToken = LHCb-EOS
         WSUrl = /srm/v2/server?SFN=
       }
-   }
+    }
 
-Note that in case of overwriting, the protocol sections are matched using the PluginName attribute, and not the section name, which is irrelevant at all.
+Note that base SE must be separated from the inherited SE in two different sections. You can also notice that the name of the protocol section can be a plugin name. In this way, you do not need to specify a plugin name inside.
```

Comment:
My experience shows that giving "meaningful" names to sections is often confusing. So, please, keep PluginName as a mandatory parameter and do not try to interpret the section name. 

Eventually (not now), I suggest to introduce yet another section, e.g. StoragePlugins, in the SE definition in which all the per plugin sections will go. Then there will be no confusion that each section corresponds to a plugin. This is how it is done for Queues in the CE sections. So, SEs should be described symmetrically. But even in this case I would still keep PluginName as mandatory. This is what was foreseen for v7r0. A crude example below:  

  StorageElements
  {
    SE1
    {
       Option1 = value
       Option2 = value
       StoragePlugins 
       {
          SomeName_1
          {
              PluginName = name1
              ...
           } 
           SomeName_2
           {
               PluginName = name 2
            }
       }
    }     
  }